lib/rfc5424.js

```diff
@@ -10,7 +10,12 @@ function messageBuilderFactory (options) {
   return function (data) {
     const severity = utils.levelToSeverity(data.level)
     const pri = (8 * options.facility) + severity
-    const tstamp = moment(data.time).tz(options.tz).format().toUpperCase()
+    const isoStr = DateTime.fromMillis(data.time, { zone: options.tz })
+      .set({ millisecond: 0 })
+      .toISO({ suppressMilliseconds: true })
+    const tstamp = isoStr.includes('+00:00')
+      ? `${isoStr.split('+')[0]}Z`
+      : isoStr
```

Comment:
Please don't strictly appease the tests. `+00:00` is the same as `Z`, but note that `2023-01-07T16:40:00-05:00` is not. The issue you are attempting to solve is the timezone representation. The default timezone for most of the tests is defined by https://github.com/pinojs/pino-syslog/blob/be6d7723c0635a427dcdd3c14d660bf072c54997/lib/utils.js#LL48C13-L48C13

See the playbook https://runkit.com/63b9e8148ba93c0008d85c98/63b9e81423d01f0008ee1a6d

Notice that the current default timezone is what is causing the incompatibility with the tests. Luxon is treating `'Etc/UTC'` (a valid string https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) differently from `'utc'` -- https://github.com/moment/luxon/blob/c12e5dc13a1d5af7ee47719dc0901117daf24c42/src/impl/zoneUtil.js#L23

I think part of the changes includes changing the default timezone from `Etc/UTC` to `utc`.

I created an upstream issue for this https://github.com/moment/luxon/issues/1358

Reply:
thank you for creating that issue i think that should solve problem.

The [above](https://github.com/pinojs/pino-syslog/pull/20/files#diff-119b09d0c1537d4f29cb9e76e8c5e509ddbfa3ee01c6c03308e49d381c346ecaR13) should print out timezone with their respective offset such as`2023-01-07T16:40:00-05:00`.

My only worry with changing default to `utc` there would be other timezones that will print `+00:00`. [here](https://gist.github.com/dancastillo/5223a3316c5488960e0425af336a582f) is example

Reply:
That is fine. The code in question just needs to generate a string according to the defined configuration.